src/components/KMultiselect/KMultiselectItem.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+  <div
+    :key="item.key"
+    class="k-multiselect-item"
+    :data-testid="`k-multiselect-item-${item.value}`"
+    @click="handleClick"
```

Comment:
suggestion: Since the `@click` event is actually on this element, could you make this outer element the `<button>` and then the interior `<button>` is just a div since it isn't actually focusable/clickable?

Reply:
Struggled with this a little. Ended up leaving the structure the same but moving the `@click` handler onto the `<button>`.